### DISCUSSION TITLE
windows 上 go install 失败？

### DISCUSSION THREAD

**Discussion by alasq** [wa-lang/wa#22] [Q&A]
![image](https://user-images.githubusercontent.com/14977888/184638059-40f78390-19c9-414a-a547-9e41273e8ed4.png)
![image](https://user-images.githubusercontent.com/14977888/184638502-0ce85543-78eb-4499-b2ab-61c45fffec2d.png)

**Comment by chai2010** [comment]
可能是网络的问题：`@master` 会拉取最新的代码，会导致读取墙外的 sum 服务器。可以改用 `github.com/wa-lang/wa@latest` 试试

**Comment by chai2010** [comment]
v0.2 已经支持 Windows 系统

**Comment by tudoutiaoya** [comment]
you can use zhe magic method (such as ladder)
